src/components/recruting/_02_prepare/_01/NumberSpinner.tsx

```
@@ -0,0 +1,68 @@
+      rules={rules}
+      defaultValue={0}
+      render={({ field: { value, onChange } }) => {
+        const increment = () => onChange(value + 1);
+        const decrement = () => {
+          if (value <= 0) return;
+          onChange(value - 1);
+        };
+
+        const handleInputChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+          const newValue = e.target.value === "" ? 0 : parseInt(e.target.value);
+          if (isNaN(newValue) || newValue < 0) {
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)